ambry-server/src/integration-test/java/com.github.ambry.server/Verifier.java

```diff
@@ -137,15 +140,33 @@ public void run() {
                     System.out.println(exceptionMsg);
                     throw new IllegalStateException(exceptionMsg);
                   }
+                  long actualExpiryTimeMs = ServerTestUtil.getExpiryTimeMs(propertyOutput);
+                  long expectedExpiryTimeMs = ServerTestUtil.getExpiryTimeMs(payload.blobProperties);
+                  if (actualExpiryTimeMs != expectedExpiryTimeMs) {
+                    String exceptionMsg =
+                        "Expiry time (props) not matching " + " expected " + expectedExpiryTimeMs + " actual "
+                            + actualExpiryTimeMs;
+                    System.out.println(exceptionMsg);
+                    throw new IllegalStateException(exceptionMsg);
+                  }
+                  actualExpiryTimeMs =
+                      resp.getPartitionResponseInfoList().get(0).getMessageInfoList().get(0).getExpirationTimeInMs();
+                  if (actualExpiryTimeMs != expectedExpiryTimeMs) {
+                    String exceptionMsg =
+                        "Expiry time (MessageInfo) not matching " + " expected " + expectedExpiryTimeMs + " actual "
+                            + actualExpiryTimeMs;
+                    System.out.println(exceptionMsg);
```

Comment:
is the println needed? I see that at a few points in this test

Reply:
I just followed whatever was already there for consistency. I think all of this can be refactored